config-caf/src/main/java/com/hpe/caf/api/CafConfigurationSource.java

```diff
@@ -257,4 +258,13 @@ protected void incrementErrors()
     {
         this.confErrors.incrementAndGet();
     }
+
+    private Method getMethod(String propertyName, Class<?> beanClass, Function<PropertyDescriptor, Method> function){
+        try{
+            PropertyDescriptor propertyDescriptor = new PropertyDescriptor(propertyName, beanClass);
+            return function.apply(propertyDescriptor);
+        } catch (IntrospectionException e) {
+            return null;
```

Comment:
Discussed change in behaviour now returning null instead of throwing.  At least if covering up fields without any accessors we should log this out in the debug logs or something.